### ISSUE TITLE
Error: serverInternal :: The server experienced an internal error :: Error creating new authz

### ISSUE THREAD

**Issue by PatrickHeppler** [letsencrypt/letsencrypt#1440]
This seems not a whitelist problem

2015-11-10 09:06:50,846:DEBUG:letsencrypt.cli:Root logging level set at 30
2015-11-10 09:06:50,847:INFO:letsencrypt.cli:Saving debug log to /var/log/letsencrypt/letsencrypt.log
2015-11-10 09:06:50,847:DEBUG:letsencrypt.cli:letsencrypt version: 0.0.0.dev20151108
2015-11-10 09:06:50,847:DEBUG:letsencrypt.cli:Arguments: ['--agree-dev-preview', '--server', 'https://acme-v01.api.letsencrypt.org/directory', '-d', 'lucee.southside-crew.club']
2015-11-10 09:06:50,847:DEBUG:letsencrypt.cli:Discovered plugins: PluginsRegistry(PluginEntryPoint#apache,PluginEntryPoint#webroot,PluginEntryPoint#null,PluginEntryPoint#manual,PluginEntryPoint#standalone)
2015-11-10 09:06:50,875:DEBUG:letsencrypt.cli:Requested authenticator None and installer None
2015-11-10 09:06:51,078:DEBUG:letsencrypt.plugins.disco:No installation (PluginEntryPoint#apache):
Traceback (most recent call last):
  File "/root/.local/share/letsencrypt/lib/python2.7/site-packages/letsencrypt/plugins/disco.py", line 103, in prepare
    self._initialized.prepare()
  File "/root/.local/share/letsencrypt/lib/python2.7/site-packages/letsencrypt_apache/configurator.py", line 144, in prepare
    raise errors.NoInstallationError
NoInstallationError
2015-11-10 09:06:51,080:DEBUG:letsencrypt.plugins.disco:Other error:(PluginEntryPoint#webroot): --webroot-path must be set
Traceback (most recent call last):
  File "/root/.local/share/letsencrypt/lib/python2.7/site-packages/letsencrypt/plugins/disco.py", line 103, in prepare
    self._initialized.prepare()
  File "/root/.local/share/letsencrypt/lib/python2.7/site-packages/letsencrypt/plugins/webroot.py", line 89, in prepare
    self.option_name("path")))
PluginError: --webroot-path must be set
2015-11-10 09:06:51,080:DEBUG:letsencrypt.display.ops:Single candidate plugin: * standalone
Description: Automatically use a temporary webserver
Interfaces: IAuthenticator, IPlugin
Entry point: standalone = letsencrypt.plugins.standalone:Authenticator
Initialized: <letsencrypt.plugins.standalone.Authenticator object at 0x29603d0>
Prep: True
2015-11-10 09:06:51,080:DEBUG:letsencrypt.cli:Selected authenticator <letsencrypt.plugins.standalone.Authenticator object at 0x29603d0> and installer None
2015-11-10 09:06:51,108:DEBUG:letsencrypt.cli:Picked account: <Account(abf61161c619f0e812db47a2bf2e5b72)>
2015-11-10 09:06:51,109:DEBUG:root:Sending GET request to https://acme-v01.api.letsencrypt.org/directory. args: (), kwargs: {}
2015-11-10 09:06:51,118:INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): acme-v01.api.letsencrypt.org
2015-11-10 09:06:51,456:DEBUG:requests.packages.urllib3.connectionpool:"GET /directory HTTP/1.1" 200 263
2015-11-10 09:06:51,495:DEBUG:root:Received <Response [200]>. Headers: {'Content-Length': '263', 'Expires': 'Tue, 10 Nov 2015 09:14:18 GMT', 'Strict-Transport-Security': 'max-age=604800', 'Server': 'nginx', 'Connection': 'keep-alive', 'Pragma': 'no-cache', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 09:14:18 GMT', 'X-Frame-Options': 'DENY', 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*', 'Replay-Nonce': 'fmT638-YMJKhmP0TfQKIdv6OnOmjyWA0vRj6g6v7Qvs'}. Content: '{"new-authz":"https://acme-v01.api.letsencrypt.org/acme/new-authz","new-cert":"https://acme-v01.api.letsencrypt.org/acme/new-cert","new-reg":"https://acme-v01.api.letsencrypt.org/acme/new-reg","revoke-cert":"https://acme-v01.api.letsencrypt.org/acme/revoke-cert"}'
2015-11-10 09:06:51,496:DEBUG:acme.client:Received response <Response [200]> (headers: {'Content-Length': '263', 'Expires': 'Tue, 10 Nov 2015 09:14:18 GMT', 'Strict-Transport-Security': 'max-age=604800', 'Server': 'nginx', 'Connection': 'keep-alive', 'Pragma': 'no-cache', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 09:14:18 GMT', 'X-Frame-Options': 'DENY', 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*', 'Replay-Nonce': 'fmT638-YMJKhmP0TfQKIdv6OnOmjyWA0vRj6g6v7Qvs'}): '{"new-authz":"https://acme-v01.api.letsencrypt.org/acme/new-authz","new-cert":"https://acme-v01.api.letsencrypt.org/acme/new-cert","new-reg":"https://acme-v01.api.letsencrypt.org/acme/new-reg","revoke-cert":"https://acme-v01.api.letsencrypt.org/acme/revoke-cert"}'
2015-11-10 09:06:52,229:INFO:letsencrypt.crypto_util:Generating key (2048 bits): /etc/letsencrypt/keys/0040_key-letsencrypt.pem
2015-11-10 09:06:52,251:INFO:letsencrypt.crypto_util:Creating CSR: /etc/letsencrypt/csr/0038_csr-letsencrypt.pem
2015-11-10 09:06:52,251:DEBUG:letsencrypt.client:CSR: CSR(file='/etc/letsencrypt/csr/0038_csr-letsencrypt.pem', data='0\x82\x02\x9f0\x82\x01\x87\x02\x000$1"0 \x06\x03U\x04\x03\x0c\x19lucee.southside-crew.club0\x82\x01"0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x01\x05\x00\x03\x82\x01\x0f\x000\x82\x01\n\x02\x82\x01\x01\x00\xad\xb2]\xae;\xe88\xb6.\xec\xcb\xfb\xbb\xeeQU\xc4\xdfz)h\x9aUG\x19v/\xbd\xa2 \xfd\x14\xb2\x14\xdd]o#-\xbc\x97\xc3ND\xcew-\x1b\xd6}\x0b6SNi\xdb\xe4\x8a[\xe48\x1d\xa3}\x99:}&\xae\x9d0[\xa7\xa9\xfe\x01\xac8\xa8@n\xc4\xb6I\xe9+4$\x9c\xa5\xba\xdd\xb2jU\x9dRbwy\x97*Q\xd7\xac*I!\x1c\xebr\x1d\xa9\x8b\x9b\x98\xc8\xbe\x16\xd5D\x0eQ\x84\xe6`M\xb7\x82\xcaA\x8bi\xec\x0b\xae\xb6\xab\x1a}\xe8\x8f\x9e\xb5\x982\xb4\xe5,(\t\x8a*rv{\xce\xc0\xb9\xd5\ty\xaecm\xac\xc1:vK\xdc\x9eC\x95[S\x02\x1a\x16\xcd\xeb\xb9F\x9e3\xdbs\xcc\xbe\xeb+k\x05\xdc\x00*\xb0.\xe4e\xb1\x85\xba;\x8bGX\x97\tZ\n\xe39\xb9~\xa4\x9d;\x8b\x1fu\xb17\xec\xa5\xb4\xa1Z\xef\xa1\xe0n\xbd?\x03\x10TX\xe8P\xda\x18\xaf\xfd\xe2\x8a\xb5\xa6\x0f\xdcte2\xc0\xa6\xcb\x02\x03\x01\x00\x01\xa0705\x06\t*\x86H\x86\xf7\r\x01\t\x0e1(0&0$\x06\x03U\x1d\x11\x04\x1d0\x1b\x82\x19lucee.southside-crew.club0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\x00\x03\x82\x01\x01\x00$\x141!_\x17X"\xea\x0f\x1dG\xde\xa3\x17\xf5\xbb>\xed\xd2\x9d\xdb@"\x95\xb5\xd8X~\xc5ti\x92\xdd,P\xef\xab\x98Y\x02\x84\x04\xf0"(B\xe0\x81U\xe6?\xd2\x08\xfa\x8c|\xdb\xcd\xd6:\x8b\xcd\x08T\xa5\x99E\xf6\xf3\xf4r\x82\xc1\xdb\xbc\x8e\x88\xdc\xea\xe1,\xc7\xfd@\x0e8\t\xe9\xde\x83\xc3!qL\xb6 \x14\xa6dr\x9c\xb4cn5\xecz\xff\xc9\xee\x11^h\xfe\x00:\xe7\x156\x8c\xaf`\x8b\x908\xaf|\xef\xf9\xdf{S\xd4\xd5\x19\xda\xc4Eu\x02\xe6\xa2\xec,\xd3\x92j\x83\x0b\xfd\x1d\xb4\x12\xe9b\xadY\xad>\xa50>\xc2\xf2F\x86\xcf\x7f\xa4b{\x8e9\xbe\x84\x06\x97VT\xa7t\x1d \xfa\xbc\xaf\xb3\x8bbT\x05\x83>\xff]\xb9D8\xaf@?\xed\x8f43\xccabTG\xd4o6\x14j,\x998q\x9f\xc6\xda\xfbu\xd6\xeb\xc5na|\x90Y\x1f~\xbe\xa0vd\xcd\x84!\xd1*@2\xb3\xc5\xdd\xf9\x16\xe4\x89<\x95\xfa', form='der'), domains: ['lucee.southside-crew.club']
2015-11-10 09:06:52,252:DEBUG:root:Requesting fresh nonce
2015-11-10 09:06:52,252:DEBUG:root:Sending HEAD request to https://acme-v01.api.letsencrypt.org/acme/new-authz. args: (), kwargs: {}
2015-11-10 09:06:52,254:INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): acme-v01.api.letsencrypt.org
2015-11-10 09:06:52,717:DEBUG:requests.packages.urllib3.connectionpool:"HEAD /acme/new-authz HTTP/1.1" 405 0
2015-11-10 09:06:52,722:DEBUG:root:Received <Response [405]>. Headers: {'Content-Length': '0', 'Pragma': 'no-cache', 'Expires': 'Tue, 10 Nov 2015 09:14:19 GMT', 'Server': 'nginx', 'Connection': 'keep-alive', 'Allow': 'POST', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 09:14:19 GMT', 'Content-Type': 'application/problem+json', 'Replay-Nonce': 'CUOHz6j4Apm10LUTGaIJ9J7sdZAgO8LbCmrB7ljPidw'}. Content: ''
2015-11-10 09:06:52,723:DEBUG:acme.client:Storing nonce: '\tC\x87\xcf\xa8\xf8\x02\x99\xb5\xd0\xb5\x13\x19\xa2\t\xf4\x9e\xecu\x90 ;\xc2\xdb\nj\xc1\xeeX\xcf\x89\xdc'
2015-11-10 09:06:52,724:DEBUG:acme.jose.json_util:Omitted empty fields: expires=None, challenges=None, combinations=None, status=None
2015-11-10 09:06:52,724:DEBUG:acme.client:Serialized JSON: {"identifier": {"type": "dns", "value": "lucee.southside-crew.club"}, "resource": "new-authz"}
2015-11-10 09:06:52,726:DEBUG:acme.jose.json_util:Omitted empty fields: x5c=(), crit=(), typ=None, jwk=None, x5u=None, kid=None, alg=None, cty=None, x5tS256=None, jku=None, x5t=None
2015-11-10 09:06:52,730:DEBUG:acme.jose.json_util:Omitted empty fields: x5c=(), crit=(), typ=None, x5u=None, kid=None, cty=None, x5tS256=None, jku=None, x5t=None, nonce=None

**Comment by Harvester57** [comment]
Same error here : 

2015-11-10 12:01:17,317:DEBUG:letsencrypt.cli:Root logging level set at 30
2015-11-10 12:01:17,321:INFO:letsencrypt.cli:Saving debug log to /var/log/letsencrypt/letsencrypt.log
2015-11-10 12:01:17,325:DEBUG:letsencrypt.cli:letsencrypt version: 0.0.0.dev20151108
2015-11-10 12:01:17,325:DEBUG:letsencrypt.cli:Arguments: ['--agree-dev-preview', '--server', 'https://acme-v01.api.letsencrypt.org/directory']
2015-11-10 12:01:17,329:DEBUG:letsencrypt.cli:Discovered plugins: PluginsRegistry(PluginEntryPoint#apache,PluginEntryPoint#webroot,PluginEntryPoint#null,PluginEntryPoint#manual,PluginEntryPoint#standalone)
2015-11-10 12:01:17,341:DEBUG:letsencrypt.cli:Requested authenticator None and installer None
2015-11-10 12:01:18,088:DEBUG:letsencrypt.plugins.disco:No installation (PluginEntryPoint#apache): 
Traceback (most recent call last):
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/plugins/disco.py", line 103, in prepare
    self._initialized.prepare()
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt_apache/configurator.py", line 144, in prepare
    raise errors.NoInstallationError
NoInstallationError
2015-11-10 12:01:18,090:DEBUG:letsencrypt.plugins.disco:Other error:(PluginEntryPoint#webroot): --webroot-path must be set
Traceback (most recent call last):
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/plugins/disco.py", line 103, in prepare
    self._initialized.prepare()
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/plugins/webroot.py", line 89, in prepare
    self.option_name("path")))
PluginError: --webroot-path must be set
2015-11-10 12:01:18,091:DEBUG:letsencrypt.display.ops:Single candidate plugin: * standalone
Description: Automatically use a temporary webserver
Interfaces: IAuthenticator, IPlugin
Entry point: standalone = letsencrypt.plugins.standalone:Authenticator
Initialized: <letsencrypt.plugins.standalone.Authenticator object at 0x7fc0cd95a050>
Prep: True
2015-11-10 12:01:18,096:DEBUG:letsencrypt.cli:Selected authenticator <letsencrypt.plugins.standalone.Authenticator object at 0x7fc0cd95a050> and installer None
2015-11-10 12:01:18,163:DEBUG:letsencrypt.cli:Picked account: <Account(88d1a5aae1a2db86cce1f5bcae40038e)>
2015-11-10 12:01:18,168:DEBUG:root:Sending GET request to https://acme-v01.api.letsencrypt.org/directory. args: (), kwargs: {}
2015-11-10 12:01:18,186:INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): acme-v01.api.letsencrypt.org
2015-11-10 12:01:18,419:DEBUG:requests.packages.urllib3.connectionpool:"GET /directory HTTP/1.1" 200 263
2015-11-10 12:01:18,426:DEBUG:root:Received <Response [200]>. Headers: {'Content-Length': '263', 'Expires': 'Tue, 10 Nov 2015 12:01:18 GMT', 'Strict-Transport-Security': 'max-age=604800', 'Server': 'nginx', 'Connection': 'keep-alive', 'Pragma': 'no-cache', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 12:01:18 GMT', 'X-Frame-Options': 'DENY', 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*', 'Replay-Nonce': 'ohM_QcKu0oMQzKI6E5qcI94Gzal0TO0cGlO59qgcWZQ'}. Content: '{"new-authz":"https://acme-v01.api.letsencrypt.org/acme/new-authz","new-cert":"https://acme-v01.api.letsencrypt.org/acme/new-cert","new-reg":"https://acme-v01.api.letsencrypt.org/acme/new-reg","revoke-cert":"https://acme-v01.api.letsencrypt.org/acme/revoke-cert"}'
2015-11-10 12:01:18,428:DEBUG:acme.client:Received response <Response [200]> (headers: {'Content-Length': '263', 'Expires': 'Tue, 10 Nov 2015 12:01:18 GMT', 'Strict-Transport-Security': 'max-age=604800', 'Server': 'nginx', 'Connection': 'keep-alive', 'Pragma': 'no-cache', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 12:01:18 GMT', 'X-Frame-Options': 'DENY', 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*', 'Replay-Nonce': 'ohM_QcKu0oMQzKI6E5qcI94Gzal0TO0cGlO59qgcWZQ'}): '{"new-authz":"https://acme-v01.api.letsencrypt.org/acme/new-authz","new-cert":"https://acme-v01.api.letsencrypt.org/acme/new-cert","new-reg":"https://acme-v01.api.letsencrypt.org/acme/new-reg","revoke-cert":"https://acme-v01.api.letsencrypt.org/acme/revoke-cert"}'
2015-11-10 12:01:18,430:DEBUG:letsencrypt.display.ops:No installer, picking names manually
2015-11-10 12:01:56,344:INFO:letsencrypt.crypto_util:Generating key (2048 bits): /etc/letsencrypt/keys/0005_key-letsencrypt.pem
2015-11-10 12:01:56,369:INFO:letsencrypt.crypto_util:Creating CSR: /etc/letsencrypt/csr/0005_csr-letsencrypt.pem
2015-11-10 12:01:56,373:DEBUG:letsencrypt.client:CSR: CSR(file='/etc/letsencrypt/csr/0005_csr-letsencrypt.pem', data='0\x82\x02\xeb0\x82\x01\xd3\x02\x01\x000\x1c1\x1a0\x18\x06\x03U\x04\x03\x0c\x11blog.harvester.fr0\x82\x01"0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x01\x05\x00\x03\x82\x01\x0f\x000\x82\x01\n\x02\x82\x01\x01\x00\xc7\xd5\xec\x90\x85\\+\x85\x1f\xd8T9\xb5k\xfc\x1a\xa3\xab\xc9r<"f\xb0z\xc98j\xdf`_\xaf\xd2\x0f\xaf\xc6\x86\xf02\x92\x0b\xa5\x1b"\xf0\xd2\x06\xda\xf27\x03\x1bw{\x851\x144%\xa7x?\x18\xa5\x118\x87\xe0\xe1\xefc+\r\xe7\xcf\xcd*\xba,[\xbf\xce\x8f\xb4\xdf\x84\xe0\rj^\xa5\xa4\x90@u\xf2\xf0t\xc24\x19\xbf\x9do\x14`tm\xc9tz\xb86\xe6\xbb\x95\\\x13*\xb2/\x05pB`8\x87\x86\n5\xcb\x96\xc3\x14o\'S\x0b\xee\x8a\xbe\x99\xa5_\x04\xe2qr\xfc\xd8h\xd5\xde\xa9oa&\xf5\xeeql\xdb\x82\xf0\x12O\x90\x14\xfbN:w\x1e\xacr+\xc4\xdb\xaea\xa6\xd8\xfd\xc7\xb2\n\x0c\x89\x9bx\xe1\x95\x18\x03\xd9\xca`\xf9r\xf6\x10\x14\xf0\x1d\xf5P\x03\xe5\xc30=\xda\xe7\xbe\xf9\xdbj\x88\x9fg\x04\x91\xd8\xfa[\xa0\x90\xb9\xa5\x85\xc7|\x02\x00\xb1L\xb21\x12Kl\xc0]Z\xfc\x816l`\r\xfa\x07\x0e\xda\xd9\x7f\x02\x03\x01\x00\x01\xa0\x81\x890\x81\x86\x06\t*\x86H\x86\xf7\r\x01\t\x0e1y0w0u\x06\x03U\x1d\x11\x04n0l\x82\x11blog.harvester.fr\x82\x0fcv.harvester.fr\x82\x11mail.harvester.fr\x82\x13reader.harvester.fr\x82\x10www.harvester.fr\x82\x0charvester.fr0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\x00\x03\x82\x01\x01\x00\xa5\x83\xc4\x18\x15Z\xa1\x8b\x90"\xbb\xb5x\xf2\xc3(H \x9c\xb20\xa6v\x8da\xcds\x9c/\xe6\r\xa6o\xd8#\x84\x13\x9e`\xf3C\xfa\x93`\xf3&]d\x17\xfb\xea{\xc7\xd9;\xf1;\xc7\xee\xef)\xeb\xf4U\xaa5\x1e\x1aZ/\xa1\x19>\xb1\x08\xeb\x1b\xe8\x85}P\xa8\x1d\xbdoU\x8c\xd80iaaz\xc4\x99g\x14OTX\xb2\x00\xa0\x9c\xc4PsJ}r\x1e\x05,D\xbeGu\xd5\x8b4\x84\xf3\xf4\x81`{\xb8\xa7\xf1\xf8\x13xj\xb1\xe5_\xc6\xf8\xb1\xab\xfb\xe5\xf6ti\x02\xcbg\xe5w\xe6\x1b\xc9\xdd\x18_q\x8d\x06\xa8\xb6\x84\x96O\xaf\xed\xb0\xe7{\x07\xf0;nV\xdfW=\xdd\xc5\xa3\xfc\xde\x07\x93Z\x1c\nE>\x1b\xf6\x82\'u\x88;\xdb\xcc@\xf8\xb2\xcb\xc8<\x14\x142@\xb68tt\x99\x93*\xdcl\xdf\x82\xd9\xe4oN|\xcf\xff\xfb\xd0k\x9f\xcd\xe0\xac\xd8\xf1\x85\\PA\x04\xa3\xa3p\x89.Lvr\xa3"3\x88\\p\xa1\x00', form='der'), domains: ['blog.harvester.fr', 'cv.harvester.fr', 'mail.harvester.fr', 'reader.harvester.fr', 'www.harvester.fr', 'harvester.fr']
2015-11-10 12:01:56,374:DEBUG:root:Requesting fresh nonce
2015-11-10 12:01:56,375:DEBUG:root:Sending HEAD request to https://acme-v01.api.letsencrypt.org/acme/new-authz. args: (), kwargs: {}
2015-11-10 12:01:56,380:INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): acme-v01.api.letsencrypt.org
2015-11-10 12:01:57,652:DEBUG:requests.packages.urllib3.connectionpool:"HEAD /acme/new-authz HTTP/1.1" 405 0
2015-11-10 12:01:57,659:DEBUG:root:Received <Response [405]>. Headers: {'Content-Length': '0', 'Pragma': 'no-cache', 'Expires': 'Tue, 10 Nov 2015 12:01:57 GMT', 'Server': 'nginx', 'Connection': 'keep-alive', 'Allow': 'POST', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 12:01:57 GMT', 'Content-Type': 'application/problem+json', 'Replay-Nonce': '7HOo07B35Dipke5ukiroVp8gvAlRUcpMsLkvMAPKK2M'}. Content: ''
2015-11-10 12:01:57,662:DEBUG:acme.client:Storing nonce: '\xecs\xa8\xd3\xb0w\xe48\xa9\x91\xeen\x92*\xe8V\x9f \xbc\tQQ\xcaL\xb0\xb9/0\x03\xca+c'
2015-11-10 12:01:57,663:DEBUG:acme.jose.json_util:Omitted empty fields: status=None, combinations=None, expires=None, challenges=None
2015-11-10 12:01:57,663:DEBUG:acme.client:Serialized JSON: {"identifier": {"type": "dns", "value": "blog.harvester.fr"}, "resource": "new-authz"}
2015-11-10 12:01:57,668:DEBUG:acme.jose.json_util:Omitted empty fields: x5c=(), crit=(), typ=None, alg=None, jku=None, cty=None, x5t=None, kid=None, x5tS256=None, x5u=None, jwk=None
2015-11-10 12:01:57,684:DEBUG:acme.jose.json_util:Omitted empty fields: x5c=(), crit=(), typ=None, jku=None, nonce=None, cty=None, x5t=None, kid=None, x5tS256=None, x5u=None
2015-11-10 12:01:57,686:DEBUG:root:Sending POST request to https://acme-v01.api.letsencrypt.org/acme/new-authz. args: (), kwargs: {'data': '{"header": {"alg": "RS256", "jwk": {"e": "AQAB", "kty": "RSA", "n": "ss30ULEXQ8rdBnfHmNVDV8tXtgLC6JmhSaIaZryuIhQi_WQK6HfzV6g8ID4O34WYgEtnsaFaJ26qAcajRy-vjHA89zNMQT_snmPQNAV2kZ7tYxTz-Ew3Cb7IDDW0_GGygm0ZPf4YiFEdCCjTMauAmMqY_ijA5DRjDXuM2HgAEALQVrTcS6GvwSRUyrr6jV_2clWaVnSi6mGDtXHRm9NeV1FlRnfnbkNNpDxZTsjl0f72FSzs_tsfIVh8rJEhVpsWDdFEKx-Gq_2hLolEsrF_OGPKKtJJiVawgF-tsfkB4eq6ZXOIdKhMvaEtcERROeAD68j2kbqZoMMB0BWx_TMakw"}}, "protected": "eyJub25jZSI6ICI3SE9vMDdCMzVEaXBrZTV1a2lyb1ZwOGd2QWxSVWNwTXNMa3ZNQVBLSzJNIn0", "payload": "eyJpZGVudGlmaWVyIjogeyJ0eXBlIjogImRucyIsICJ2YWx1ZSI6ICJibG9nLmhhcnZlc3Rlci5mciJ9LCAicmVzb3VyY2UiOiAibmV3LWF1dGh6In0", "signature": "pM1hsaxa5VySc0DdmvEnRByakjMDnJTiVBUhCm60HI_z8YsUMiIOkqDqHC_jU4xxOUs0LcMDw_h_jAEWV7QT8iXIcqp44AObvGbTYwUywYJDeCRbOuF_oOSgenyMB3dI8pX-CPJs1cd9hPIVAHqEtuVng78fXsw2CLAK7Mjn8LFYRspN3fNvbPFlWAXPV-NI7xRD5W1ZQBMEa9PBdWPSawOY7aOfQ40ai-DY3zjnpuyIsetJaUpCE6rnTAes_-HrN6nu3AiB75bnesiGhY7IznsN_lJKwrbTLqQr1IGTaUwLgq1lBuYD7HPPXOc_bj98AlZl5_eSuUDpNV853DvAhw"}'}
2015-11-10 12:01:57,690:INFO:requests.packages.urllib3.connectionpool:Starting new HTTPS connection (1): acme-v01.api.letsencrypt.org
2015-11-10 12:02:00,357:DEBUG:requests.packages.urllib3.connectionpool:"POST /acme/new-authz HTTP/1.1" 500 76
2015-11-10 12:02:00,363:DEBUG:root:Received <Response [500]>. Headers: {'Content-Length': '76', 'Expires': 'Tue, 10 Nov 2015 12:02:00 GMT', 'Server': 'nginx', 'Connection': 'close', 'Pragma': 'no-cache', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 12:02:00 GMT', 'Content-Type': 'application/problem+json', 'Replay-Nonce': '81yox3DNqDSvdqokD5jb1pdx-wDj8v3R8gUo71ybCrk'}. Content: '{"type":"urn:acme:error:serverInternal","detail":"Error creating new authz"}'
2015-11-10 12:02:00,366:DEBUG:acme.client:Storing nonce: '\xf3\\\xa8\xc7p\xcd\xa84\xafv\xaa$\x0f\x98\xdb\xd6\x97q\xfb\x00\xe3\xf2\xfd\xd1\xf2\x05(\xef\\\x9b\n\xb9'
2015-11-10 12:02:00,367:DEBUG:acme.client:Received response <Response [500]> (headers: {'Content-Length': '76', 'Expires': 'Tue, 10 Nov 2015 12:02:00 GMT', 'Server': 'nginx', 'Connection': 'close', 'Pragma': 'no-cache', 'Cache-Control': 'max-age=0, no-cache, no-store', 'Date': 'Tue, 10 Nov 2015 12:02:00 GMT', 'Content-Type': 'application/problem+json', 'Replay-Nonce': '81yox3DNqDSvdqokD5jb1pdx-wDj8v3R8gUo71ybCrk'}): '{"type":"urn:acme:error:serverInternal","detail":"Error creating new authz"}'
2015-11-10 12:02:00,377:DEBUG:letsencrypt.cli:Exiting abnormally:
Traceback (most recent call last):
  File "/root/.local/share/letsencrypt/bin/letsencrypt", line 11, in <module>
    sys.exit(main())
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/cli.py", line 1140, in main
    return args.func(args, config, plugins)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/cli.py", line 490, in obtaincert
    _auth_from_domains(le_client, config, domains, plugins)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/cli.py", line 315, in _auth_from_domains
    new_certr, new_chain, new_key, _ = le_client.obtain_certificate(domains)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/client.py", line 212, in obtain_certificate
    return self._obtain_certificate(domains, csr) + (key, csr)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/client.py", line 170, in _obtain_certificate
    authzr = self.auth_handler.get_authorizations(domains)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/letsencrypt/auth_handler.py", line 74, in get_authorizations
    domain, self.account.regr.new_authzr_uri)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/acme/client.py", line 215, in request_domain_challenges
    typ=messages.IDENTIFIER_FQDN, value=domain), new_authz_uri)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/acme/client.py", line 195, in request_challenges
    response = self.net.post(new_authzr_uri, new_authz)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/acme/client.py", line 628, in post
    return self._check_response(response, content_type=content_type)
  File "/root/.local/share/letsencrypt/local/lib/python2.7/site-packages/acme/client.py", line 544, in _check_response
    raise messages.Error.from_json(jobj)
Error: serverInternal :: The server experienced an internal error :: Error creating new authz

**Comment by bmw** [comment]
Thanks for posting your logs @PatrickHeppler and @Harvester57. Since #1327 is about the same problem, I'm closing this issue. I referenced this issue there so we can use your logs for debugging.

**Comment by Harvester57** [comment]
Will just copy-paste what I wrote earlier in the previous thread : 

"

I would like to add that I could create certificates last week, without issues, and this error arose when I tried to re-generate them while adding some of my subdomains (which where whitelisted after my first certificates were generated).

To be precise, I successfully generated a certificate for "harvester.fr www.harvester.fr" last week, was working fine, and when "blog.harvester.fr reader.harvester.fr mail.harvester.fr" where whitelisted, I came across this error. And, the worst part, I can't regenerate certificates for "harvester.fr www.harvester.fr" anymore !

I read on other posts that it was due to DNS misconfiguration (CNAME vs. A) and I would like to point that all my subdomains are advertised using A. And that my DNS configuration didn't change during the week, so a configuration problem is very unlikely the culprit.

Hope that helps."
